system/shaders/GL/1.2/gl_convolution-4x4.glsl

```diff
@@ -25,13 +25,9 @@ uniform float m_alpha;
 varying vec2      m_cord;
 uniform sampler1D kernelTex;
 
-//nvidia's half is a 16 bit float and can bring some speed improvements
-//without affecting quality
-#ifndef __GLSL_CG_DATA_TYPES
-  #define half float
-  #define half3 vec3
-  #define half4 vec4
-#endif
+#define half float
```

Comment:
why don't you remove the ancient NVidia types completely?

Reply:
"Keep the defines as else the lines to be changed would be way more, it does not harm." <- laziness :-)

Reply:
I removed it completely :-)

Reply:
hehe, empathy with folks, who will look at this in the future and wonder where those strange defines come from, won against laziness :)